src/components/commons/Modal.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState, useEffect, Dispatch, SetStateAction, useRef, forwardRef } from "react";
+import { Swiper, SwiperSlide } from "swiper/react";
+import "swiper/css";
+import { Swiper as SwiperType } from "swiper"; // Swiper 타입 임포트
+import { ModalWrapper, ModalBox, ModalCloseButton, ModalItemWrapper, ModalVideoWrapper, ToonWrapper, Pagination, ModalControl } from "@styles/commons/modal.style";
+interface ModalProps {
+    type: string;
+    itemId: number;
+    modalOpen: boolean;
+    setModalOpen: Dispatch<SetStateAction<boolean>>;
+}
```

Comment:
```suggestion
import React, { useState, useEffect, Dispatch, SetStateAction, useRef, forwardRef } from "react";
import { Swiper, SwiperSlide } from "swiper/react";
import "swiper/css";

import { ModalWrapper, ModalBox, ModalCloseButton, ModalItemWrapper, ModalVideoWrapper, ToonWrapper, Pagination, ModalControl } from "@styles/commons/modal.style";

import type { Swiper as SwiperType } from "swiper"; // Swiper 타입 임포트

interface ModalProps {
    type: string;
    itemId: number;
    modalOpen: boolean;
    setModalOpen: Dispatch<SetStateAction<boolean>>;
}
```

`import`문을 보기 편하도록 줄바꿈을 추가하면 좋겠습니다. 또한 타입의 경우에는 `type`을 추가하여 다른 변수명과 구분하면 좋겠습니다. 저는 주로 다음과 같은 구조를 사용합니다.
```
import ... // 라이브러리 및 프레임워크로부터 import 문

import ... // 프로젝트 구조 내에서 import 문

import type ... // 타입 import 문
```